papaya-core/src/main/java/pl/edu/agh/papaya/service/project/ProjectCreationWizard.java

```diff
@@ -45,11 +46,20 @@ public ProjectCreationWizard withChannelName(String channelName) {
         return this;
     }
 
+    public ProjectCreationWizard withJiraUrl(String jiraUrl) {
+        if (!new UrlValidator().isValid(jiraUrl)) {
```

Comment:
Ta walidacja w sumie nie jest idealna, bo jak nie ma `http[s]` na początku, to zwraca `false`